### PR TITLE
feat(dms): add support for document merge service 3.0.0

### PR DESCRIPTION
**Pull request by winpat**
`UNOCONV_LOCAL` is no longer supported in DMS 3.0.0.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -412,13 +412,10 @@ document_merge_service_oidc_userinfo_endpoint: "{{ camac_auth_idp_url }}/realms/
 document_merge_service_git_repository_url: "https://github.com/adfinis-sygroup/document-merge-service/"
 
 # Document merge service git repository
-document_merge_service_version: "v2.0.0"
+document_merge_service_version: 'v3.0.0'
```
Comment (anehx) [inline defaults/main.yml]: Double quotes :wink: Use prettier for yml!

File: templates/unoconv.service.j2 [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Unoconv listener for document conversions
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/python3 /usr/bin/unoconv --listener -p 2002
```
Comment (anehx) [inline templates/unoconv.service.j2]: Is that port configured somewhere or is it the default?

**Review by anehx — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (winpat) [reply defaults/main.yml]: Fixed. Will do in the future.

File: templates/unoconv.service.j2 (hunk 2 above)
Comment (winpat) [reply templates/unoconv.service.j2]: Yes, 2002 is the default port. I removed the `-p` flag.

**Review by anehx — APPROVED** [review]

**Pull request merged**